webapp/ext/controller/ListReportExt.controller.js

```diff
@@ -0,0 +1,69 @@
+sap.ui.define([
+    "sap/ui/model/Filter", 
+    "sap/ui/comp/smartfilterbar/SmartFilterBar", 
+    "sap/m/DateRangeSelection",
+	"../mixin/functionImports"
+], function (Filter,
+	SmartFilterBar,
+	DateRangeSelection,
+    functionImports) {
+    "use strict";
+    return {
+        getCustomAppStateDataExtension: function (oCustomData) {
+            if (oCustomData) {
+                const oCustomFieldDate = this.oView.byId("CreateAtInput");
+                if (oCustomFieldDate) {
+                    oCustomData.ReleaseDate = {
+                        startDate: oCustomFieldDate.getDateValue(),
+                        endDate: oCustomFieldDate.getSecondDateValue()
+                    };
+                }
+            }
+        },
+        restoreCustomAppStateDataExtension: function (oCustomData) {
+            if (oCustomData) {
```

Comment:
You could simplify condition
```suggestion
            if (oCustomData?.ReleaseDate) {
```